ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -363,4 +392,47 @@ public void testListTablesStatement() throws Exception {
 
     assertEquals(expectedTable, testTables.get(0));
   }
+
+  @Test
+  public void shouldFailForIncorrectCSASStatementResultType() throws Exception {
+    KsqlResource testResource = TestKsqlResourceUtil.get();
+    String ksqlString1 = "CREATE STREAM s1 AS SELECT * FROM test_table;";
+
+    Response response1 = testResource.handleKsqlStatements(new KsqlRequest(ksqlString1, Collections
```

Comment:
Here and elsewhere in this class, we can use `Collections.emptyMap()` instead and then we will not have the unchecked warnings

Reply:
Done.